axlearn/common/ssm.py

```diff
@@ -882,6 +883,30 @@ def _conv_update(
             conv_state = conv_state + bias
         return conv_state, new_cache
 
+
+    def _full_sequence_forward(
```

Comment:
Looks to me like this function duplicates the function on line 719. We don't need this again, right?